### PR TITLE
Update all of url information from "01org" to "intel"

### PR DESCRIPTION
**Pull request by wanghongjuan**
- Update all of url from https://github.com/01org/zephyrjs-ide to https://github.com/intel/zephyrjs-ide

### CODE REVIEW

**Comment by wanghongjuan** [comment]
Does the travis ci build failed related to this repository url changes?

File: src/client/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" href="<%= APP_BASE %>assets/favicon/favicon-<%= ENV_CONFIG.ENV %>.ico" type="image/x-icon">
 
-  <!-- Origin Trial Token, feature = WebUSB (For Chrome M57+), origin = https://01org.github.io, expires = 2017-09-05 -->
+  <!-- Origin Trial Token, feature = WebUSB (For Chrome M57+), origin = https://intel.github.io, expires = 2017-09-05 -->
```
Comment (nagineni) [inline src/client/index.html]: This token was expired on 2017-09-05 and the WebUSB origin trial ended on the same date. So, we can remove this token.

Note that the WebUSB feature shipped in Chrome M61 by default.

Comment (wanghongjuan) [reply src/client/index.html]: Thanks for your comment, I just removed the token.

File: src/client/index.html [hunk 2]
```diff
@@ -9,8 +9,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" href="<%= APP_BASE %>assets/favicon/favicon-<%= ENV_CONFIG.ENV %>.ico" type="image/x-icon">
 
-  <!-- Origin Trial Token, feature = WebUSB (For Chrome M57+), origin = https://01org.github.io, expires = 2017-09-05 -->
-  <meta http-equiv="origin-trial" data-feature="WebUSB (For Chrome M57+)" data-expires="2017-09-05" content="As8y4LG/sAALCqUjzVvTYkwhhnCfeVCCcI28e4j6WoqPuc0/45BJzx4oUkqzzj/8tcWzFW+Hp6yrIyiivY7y8QkAAABQeyJvcmlnaW4iOiJodHRwczovLzAxb3JnLmdpdGh1Yi5pbzo0NDMiLCJmZWF0dXJlIjoiV2ViVVNCMiIsImV4cGlyeSI6MTUwNDU2OTYwMH0=">
+  <!-- Origin Trial Token, feature = WebUSB (For Chrome M57+), origin = https://intel.github.io, expires = 2017-09-05 -->
```
Comment (nagineni) [inline src/client/index.html]: Please remove this line also.

**Comment by brianjjones** [comment]
@nagineni @wanghongjuan 
Is https://github.com/intel/zephyrjs-ide the correct URL? I just get the github page when I go there rather than the IDE.

File: src/client/index.html (hunk 2 above)
Comment (wanghongjuan) [reply src/client/index.html]: Removed

**Comment by wanghongjuan** [comment]
@brianjjones This repo was migrated from 01org to Intel recently, @nagineni updated this information on [PR#1672](https://github.com/01org/zephyr.js/pull/1672) in Zephyr.js last week.

**Comment by brianjjones** [comment]
@wanghongjuan I know it got updated, but my question is it the right URL?  Because if I just paste the URL into my browser I get the github page, not the IDE.  Previously if you pasted the URL in your browser it would go to the IDE.

**Comment by nagineni** [comment]
Soure repo URL: https://github.com/intel/zephyrjs-ide
IDE URL: https://intel.github.io/zephyrjs-ide/

This patch has the references to the source repo's location, so I would say the patch is okay. 

For some reason mac build is failing, and that needs to addressed.

**Comment by brianjjones** [comment]
@nagineni Strange the other day my device sent me to github.com/intel/zephyrjs-ide, which is why I thought the link was wrong.  Seems OK now with  https://intel.github.io/zephyrjs-ide.  I'll talk with Jimmy to see what is going on with the mac build.

**Comment by brianjjones** [comment]
Update: I've found that the e2e.ci test is hitting a snag because some of the IDE's elements have changed, namely the edit-tab.  However, these things didn't change recently so I'm guessing something about angular / e2e was different and it allowed for these errors to pass.  Anyway, I'm making progress and hope to have a fix in today.

**Comment by brianjjones** [comment]
@nagineni I still haven't been able to get it to build on Mac I'm afraid.  It appears the Protractor command sendKeys("NEW TITLE") isn't working, it doesn't update the field.  I can erase the text, but I can't put anything new in it.  I'll keep digging but I'm new to Protractor, angular, etc so its slow going.  Let me know if you have any insight :)

@grgustaf The only thing broken here is the test.  If I run the IDE on the Mac it works fine.  I'm tempted to just accept the PR so that we can get things updated with the new URL. Thoughts?

Thanks all.

**Comment by brianjjones** [comment]
@nagineni  I think I finally got the correct package combo to get the Mac build working again, see #204 .
I need to head out before it finishes on Travis, but it built on my local Mac so hopefully it takes care of the issue.  Once that's in we can accept this one too (provided it passes the tests with the update)

**Pull request merged**